mmv1/third_party/terraform/services/sql/resource_sql_database_instance_test.go

```diff
@@ -2044,6 +2044,31 @@ func TestAccSqlDatabaseInstance_rootPasswordShouldBeUpdatable(t *testing.T) {
 	})
 }
 
+func TestAccSqlDatabaseInstance_SqlServerTimezone(t *testing.T) {
```

Comment:
This test only checks the creation/deletion of the resource. https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test should cover more details on update tests. Please let me know if you have any questions adding the tests